packages/fast-components-styles-msft/src/caption/index.ts

```diff
@@ -0,0 +1,11 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ICaptionClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<ICaptionClassNameContract, IDesignSystem> = {
+    caption: {},
```

Comment:
Just verifying there are no additional styles to add that typography does not already add? (e.g. color?)